src/pages/leadership/_leaders.js

```diff
@@ -309,13 +329,17 @@ const Leaders = () => {
                 <Carousel {...settings}>
                     {leaders_data.map((leader, idx) => (
                         <LeaderMobile key={idx}>
-                            <QueryImage data={data[`${leader.image}_mobile`]} alt={leader.name} />
+                            <ImageWrapperMobile>
+                                <ImageMobileQueryImgWrapper>
+                                    <QueryImage data={data[`${leader.image}`]} alt={leader.name} />
+                                </ImageMobileQueryImgWrapper>
+                            </ImageWrapperMobile>
                             <Header size="24px" align="center" m="16px 0 0">
```

Comment:
```suggestion
                            <Header size="24px" as="h4" align="center" m="16px 0 0">
```

Reply:
Let's set this to h4 to improve our semantics.

Reply:
changed in the latest commit